### PR TITLE
windoes otel app - adding security log related dashboard doc update

### PR DESCRIPTION
**Pull request by sumoanema**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md [hunk 1]
```diff
@@ -174,6 +176,17 @@ Use this dashboard to:
 
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Windows-OpenTelemetry/Windows-Default.png' alt="Windows - Default" />
 
+### Windows - Login Status
+
+The **Windows - Login Status** dashboard provides information about successful and failed logins, successful Remote Desktop Protocol (RDP) reconnects, and failed login outliers.
+
+Use this dashboard to:
+* Monitor Successful and Failed logins by the user and track their locations with successful and failed login attempts.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md:184]:
```suggestion
* Monitor successful and failed logins by the user and track their locations with successful and failed login attempts.
```

File: docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md [hunk 2]
```diff
@@ -174,6 +176,17 @@ Use this dashboard to:
...
+* Monitor RDP (Remote Desktop) reconnect events.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md:185]:
```suggestion
* Monitor RDP reconnect events.
```

File: docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md [hunk 3]
```diff
@@ -174,6 +176,17 @@ Use this dashboard to:
...
+* Track if there are failed login outliers to identify mischievous login activities.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md:186]:
```suggestion
* Track failed login outliers to identify mischievous login activities.
```

**Comment by sumoanema** [comment]
@JV0812 applied the suggestion.  Could you please review.

**Review by JV0812 — APPROVED** [review]

File: docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md [hunk 4]
```diff
@@ -151,13 +151,15 @@ This sample log query is from the **Windows - Overview** dashboard > **System Re
 
 ### Windows - Overview
 
-The **Windows - Overview** dashboard provides insights into fatal or warning messages, policy changes, and system restarts.
+The **Windows - Overview** dashboard provides insights into fatal or warning messages, policy changes system restarts and changes to administrative groups.
```
Comment (kimsauce) [inline docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md:154]:
```suggestion
The **Windows - Overview** dashboard provides insights into fatal or warning messages, policy changes, system restarts, and changes to administrative groups.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fix

File: docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ The Windows App, which is based on the Windows event log format, consists of pre
 
 ## Fields Created in Sumo Logic for Windows
 
-Following are the [fields](/docs/manage/fields/) which will be created as part of Windows App install if not already present. 
+Following are the [fields](/docs/manage/fields/) which will be created as part of Windows App install if not already present.
```
Comment (kimsauce) [inline docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md:25]:
```suggestion
Following are the [fields](/docs/manage/fields/) that will be created as part of Windows App install if not already present. 
```

**Review by kimsauce — APPROVED** [review]

**Comment by JV0812** [comment]
The intent of this PR is covered i PR: https://github.com/SumoLogic/sumologic-documentation/pull/2705. So closing this PR.

**Comment by jpipkin1** [comment]
@JV0812, since you closed this PR with unmerged commits, you may want to delete the branch. (Scroll down to the "Closed with unmerged commits" box and click "Delete branch".)

**Pull request closed without merging**